gulpfile.js

```diff
@@ -17,74 +17,94 @@ var gulp = require('gulp'),
 // picking up dependencies of the primary entry points and putting any
 // limitations on directory structure for entry points.
 var sources = {
-    builds: ['js/detail.js'],
-    core: [
-        'js/readthedocs-doc-embed.js',
-        'js/autocomplete.js',
-        'js/projectimport.js',
-    ],
-    projects: ['js/tools.js'],
-    gold: ['js/gold.js'],
-    donate: ['js/donate.js']
+    builds: {'js/detail.js': {}},
+    core: {
+        'js/readthedocs-doc-embed.js': {create_module: false},
+        'js/autocomplete.js': {},
+        'js/projectimport.js': {},
+    },
+    projects: {'js/tools.js': {}},
+    gold: {'js/gold.js': {}},
+    donate: {'js/donate.js': {}}
 };
 
 // Standalone application to create vendor bundles for. These can be imported
 // with require in the browser or with Node during testing.
-var standalone = ['jquery', 'knockout'];
+var standalone = {
+    'jquery': {standalone: 'jquery'},
+    'knockout': {},
```

Comment:
Standalone is a library that functions like the legacy libraries, you can't import via `require()`. Ideally, we'd have one library that could do both, but browserify doesn't create libraries like it suggests.

With a patch to sphinx-ext, these libraries will only ever be used by stale documentation. A patch to sphinx-ext and sphinx-theme will generate libraries that will point to the new libraries that are available by `require()`, or if it makes sense, bundled all together.

I'm playing more with the external javascript pieces today to see what makes the most sense.

Regarding knockoutjs, it is a new dependency and should never be used in the legacy context, always via `require()`.